### PR TITLE
#2 Implement business logic for processing chunk messages

### PR DESCRIPTION
**Pull request by NorbertNagySeres**

### CODE REVIEW

File: src/main/java/com/fractal/collectorservice/infrastructure/api/controller/ChunkController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    private final ChunkRequestHandler chunkRequestHandler;
+
+    public Mono<ServerResponse> getChunks(ServerRequest serverRequest) {
+        return chunkRequestHandler.getChunksByCorrelationId(serverRequest.pathVariable(CORRELATION_ID))
+                .collectList()
+                .flatMap(chunks -> ServerResponse
+                        .ok()
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .bodyValue(chunks))
+                ;
```
Comment (szabozsoltbors) [inline src/main/java/com/fractal/collectorservice/infrastructure/api/controller/ChunkController.java:26]: I would not place this character into a separate line.

**Comment by szabozsoltbors** [comment]
Could you extend please this Postman collection with an example, how the collectors API can be used: https://github.com/azure-fractal/postman

File: src/main/java/com/fractal/collectorservice/infrastructure/api/controller/ChunkController.java (hunk 1 above)
Comment (NorbertNagySeres) [reply src/main/java/com/fractal/collectorservice/infrastructure/api/controller/ChunkController.java]: Fixed

**Comment by szabozsoltbors** [comment]
Nice and clean code!
Congratulations!

**Pull request merged**